definitions/checks/foreman_tasks/not_paused.rb

```diff
@@ -5,6 +5,7 @@ class NotPaused < ForemanMaintain::Check
       for_feature :foreman_tasks
       description 'check for paused tasks'
       tags :default
+      after :disk_io, :verify_self_registration
```

Comment:
I thought of keeping the checks for paused and running tasks together. These would collectively run before task-cleanup checks.
This was just to keep it understandable while execution.
`before :foreman_tasks_not_running` would do? or should get rid of it altogether?